mono/mini/interp/interp.c

```diff
@@ -5215,7 +5215,7 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 
 			memcpy (addr, sp [-1].data.vt, i32);
 			vt_sp -= ALIGN_TO (i32, MINT_VT_ALIGNMENT);
-			ip += 4;
+			ip += 5;
```

Comment:
how did this ever work?

Reply:
I think the instruction stream is alloc0 or something, so it would just execute a MINT_NOP (which has opcode 0), which just does the missing ip++. :D